python/ql/test/library-tests/web/zope/options

```diff
@@ -0,0 +1 @@
+semmle-extractor-options: --max-import-depth=3 -p ../../../query-tests/Security/lib/ --respect-init=False
```

Comment:
We removed `--respect-init=False` when moving the `twisted` tests. I'm wondering if it could be removed here as well, without affecting the tests.

Reply:
Good catch. I'm quite sure I was looking at some zope source code and noticed that they didn't have an `zope/__init__.py` file, which I why I did it like this. However, checking their [codebase](https://github.com/zopefoundation/zope.interface/tree/master/src/zope) now I can see that they do in fact have a `zope/__init__.py` file. Fixed it up, and tests passed locally :+1: